app/grandchallenge/pages/views.py

```diff
@@ -78,8 +98,21 @@ def get_object(self, queryset=None):
         return page
 
 
+class ChallengeSettingsBase(
+    LoginRequiredMixin, ObjectPermissionRequiredMixin, TemplateView
+):
+    template_name = "pages/challenge_settings_base.html"
+    permission_required = "change_challenge"
+    raise_exception = True
+    login_url = reverse_lazy("account_login")
+
+    def get_permission_object(self):
+        return self.request.challenge
```

Comment:
Do we need this view and URL? It seems to only show the settings sidebar. 

If we do need it, then I think we need a test for the permissions at least.

Reply:
Ah yes, you're right, we do not need this view anymore. I will remove it and also adjust the tests accordingly